pkg/helpers/filesaver_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+package helpers
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/Azure/aks-engine/pkg/i18n"
+)
+
+func TestFileSaver(t *testing.T) {
+
+	tmpDir, err := ioutil.TempDir("", "filesavertest")
+	if err != nil {
+		t.Error(err)
+	}
+	defer os.RemoveAll(tmpDir)
+	tmpFile, err := ioutil.TempFile("", "filesavertest")
+	if err != nil {
+		t.Error(err)
+	}
+	defer os.Remove(tmpFile.Name())
+
+	type SaveFileEntry struct {
+		Dir  string
+		File string
+		Data string
+	}
+	saveFileEntries := []SaveFileEntry{
+		{
+			Dir:  "", // test that an empty dir doesn't error
+			File: tmpFile.Name(),
+			Data: "",
+		},
+		{
+			Dir:  tmpDir,
+			File: "ubernetes.json",
```

Comment:
missing "k" intentional or typo?

Reply:
It was intentional since it's just a test file, but probably unnecessarily confusing in retrospect.